patterns/common/sailfish-device-capabilities.yaml.template

```diff
@@ -0,0 +1,8 @@
+Description: All available capabilities. (Template file lists all, final .yaml will reflect only what device needs)
+Name: sailfish-device-capabilities
+Requires:
+%have_vibrator- pattern:sailfish-ha-vibrator
```

Comment:
we need a generic .yaml (not per-device) for capabilities. we had one (jolla-configuration-sfe) but nuked in this PR (renamed to porter-tools)

adding to @DEVICE@ wouldn't update existing devices - existing devices would need to update not only their .specs in case a new %have_featurex comes in, but also each device's .yaml file